### PR TITLE
Caesars Palace footer #8

### LINKED ISSUE #8: Caesars Palace footer
Footer to be built from footer.docx in Franklin

footer

### PR DESCRIPTION
**Pull request by arumsey**
Fix #8 

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/
- After: https://issue-8-footer--caesars--hlxsites.hlx.page/caesars-palace/

## 📝 Description:
  
- add mobile/tablet layout
- add non-mobile layout
- add social icons

### Desktop
image

### Mobile/Tablet
image

### CODE REVIEW

File: caesars-palace/blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,238 @@
...
+}
+
+footer .footer .footer-property-social a {
+  display: flex;
 }
 
-footer .footer p {
+footer .footer .footer-property-social svg {
+  font-size: 32px;
+  width: 16px;
```
Comment (tmathern) [inline caesars-palace/blocks/footer/footer.css:85]: Use a variable like `--layout-size-m`?

File: caesars-palace/blocks/footer/footer.css [hunk 2]
```diff
@@ -1,14 +1,238 @@
...
+  height: 16px;
```
Comment (tmathern) [inline caesars-palace/blocks/footer/footer.css:86]: Use a variable like `--layout-size-m`?

File: caesars-palace/blocks/footer/footer.css [hunk 3]
```diff
@@ -1,14 +1,238 @@
...
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+  width: 100%;
+  cursor: pointer;
+  overflow: visible;
+}
+
+footer .footer .footer-property-links button svg {
+  width: 16px;
```
Comment (tmathern) [inline caesars-palace/blocks/footer/footer.css:140]: Use a variable like `--layout-size-m`?

File: caesars-palace/blocks/footer/footer.css [hunk 4]
```diff
@@ -1,14 +1,238 @@
 footer {
-  padding: 2rem;
-  background-color: var(--static-color-gray);
+  --text-color: var(--static-color-white);
```
[the remark this answers is not in the document]
Comment (arumsey) [reply caesars-palace/blocks/footer/footer.css:2]: Good catch! The font is different.

**Review by amol-anand — APPROVED** [review]

**Pull request merged**